ansible/README_DISTRIBUTED.md

```diff
@@ -0,0 +1,110 @@
+Deploying Distributed OpenWhisk using Ansible
+=========
+
+### Distributed Deployment using Ansible
+**Caveat:** All Ansible commands are meant to be executed from the `ansible` directory.
+This is important because that's where `ansible.cfg` is located which contains generic settings that are needed for the remaining steps.
+
+By default, if you omit the `-i` parameter in ansible commands, the `local` environment will be used.
+
+In all instructions, replace `<openwhisk_home>` with the base directory of your OpenWhisk source tree. e.g. `openwhisk`
+
+Login to your bootsrapper VM. Your local machine can act as bootstrapper as well, if it can connect to other VMs.
+
+Install prereqs for ansible
+```
+sudo apt-get -y install python-pip python-setuptools python-dev libssl-dev
+```
+
+Install Ansible and cloud module packages:
+```
+sudo pip install ansible==2.0.2.0 markupsafe shade pitz positional appdirs monotonic rfc3986 jsonschema
```

Comment:
In previous tests, I've had to explicitly install these dependencies for shade to install and import properly. I just stood up a new box and got this error as a result of "pip install shade"
```
error in setup command: Error parsing /tmp/pip_build_root/positional/setup.cfg: ImportError: No module named pytz
```

I've also seen similar problems where shade installs, but fails to be loaded due to an importerror of the other dependencies listed. This may be a problem with the version of pip that's currently in the apt repos.